src/graphql/network.rs

```diff
@@ -126,23 +126,57 @@ impl RawEventFilter for SearchFilter {
     }
 }
 
+/// Represents an event extracted from a session.
 #[derive(SimpleObject, Debug, ConvertGraphQLEdgesNode)]
 #[graphql_client_type(names = [conn_raw_events::ConnRawEventsConnRawEventsEdgesNode, network_raw_events::NetworkRawEventsNetworkRawEventsEdgesNodeOnConnRawEvent])]
 struct ConnRawEvent {
+    /// TODO: Waiting for an answer in the general repository.
     timestamp: DateTime<Utc>,
+    /// Source IP, or source IP address in long format.
```

Comment:
`Source IP address ` only would suffice without `Source IP`

Reply:
What does `long format` means?

Reply:
It is now modified to `Source IP address.`. Long format was what I got from consultation with ChatGPT. I don't think we need it anymore.